### PR TITLE
Improved css

### PR DESCRIPTION
**Pull request by Lilykhan786**

### CODE REVIEW

File: .replit [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language = "nodejs"
```
Comment (kartikcho) [inline .replit]: Could you remove this file from the PR as it isn't necessary to the project

File: src/globals.css [hunk 2]
```diff
@@ -272,10 +273,14 @@ body {
   border: none;
   width: 60%;
   display: block;
-  border-bottom: 0.3rem solid transparent;
+  /* border-bottom: 0.3rem solid transparent; */
```
Comment (kartikcho) [inline src/globals.css]: Remove this line instead of pushing the comment

**Review by kartikcho — CHANGES_REQUESTED** [review]
Everything looks great, minor nits and we're good!

File: src/globals.css (hunk 2 above)
Comment (Lilykhan786) [reply src/globals.css]: done

**Comment by kartikcho** [comment]
LGTM!

**Pull request merged**